test/support/hello_world/hello-world.js

```diff
@@ -0,0 +1,15 @@
+export function greet(who) {
+  return `Hello, ${who}!`;
+}
+
+export function multiGreet(who, times) {
+  var results = [];
+  for(let i = 0; i < times; i++) {
+    results.push(greet(who));
+  }
+  return results;
+}
+
+export function greetMany(people) {
+  return people.map((person) => `Hello, ${person}!`);
+}
```

Comment:
oh we have js now! what's the purpose of this file?

Reply:
ah wait. this is the source code of the hello-world wasm component?!

Reply:
yes :)